etcd-manager/pkg/volumes/hetzner/volumes.go

```diff
@@ -230,6 +230,9 @@ func getMatchingVolumes(client *hcloud.Client, matchLabels map[string]string) ([
 			labels = append(labels, k+"="+v)
 		}
 	}
+	if nameLabel != "" {
```

Comment:
Should this logic be inside or outside of the function? I think I would expect not to have the extra function argument, but rather add the nameLabel to matchLabels.

Reply:
How about now?